.github/workflows/cloud-deploy-embed.yml

```diff
@@ -0,0 +1,51 @@
+name: Deploy Embed to Netlify
+
+# Controls when the action will run. Triggers the workflow on push or pull request
+on:
+  workflow_call:
+    inputs:
+      environment:
+        required: true
+        type: string
+
+jobs:
+  reusable_embed_deploy:
+    runs-on: ubuntu-latest
+    timeout-minutes: 80
+    environment: ${{ inputs.environment }}
+    permissions:
+      contents: read
+      packages: write
+      deployments: write
+      id-token: write
+    steps:
+      - uses: actions/checkout@v4
+      - uses: ./.github/actions/setup-project
+        with:
+          slim: 'true'
+
+      - name: Build
```

Comment:
❓ Do we need both build:embed and build:prod steps? Are they doing different things? cc: @LetItRock